portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -43,6 +43,13 @@ definition {
 			locator1 = "FormFields#GRID_OPTIONS_OPTION_CHECKED");
 	}
 
+	macro assertHelpTextPresent {
+		AssertTextEquals(
+			key_fieldName = "${fieldName}",
```

Comment:
@leilanyaragao, apparently, this is not needed, after all the path `FormFields#HELP_TEXT_VALUE` isn't using it. Can you please remove?